Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -3001,7 +3037,7 @@ int CLuaGUIDefs::GUIEditSetReadOnly(lua_State* luaVM)
 
 int CLuaGUIDefs::GUIEditIsReadOnly(lua_State* luaVM)
 {
-    // bool guiEditIsReadOnly( element editField )
```

Comment:
Unneeded changes. You can fix this by copying over this file from master, and then reapplying your additions in this file.